systemvm/patches/debian/vpn/etc/ipsec.d/l2tp.conf

```diff
@@ -30,4 +33,6 @@ conn L2TP-PSK
         # ----------------------------------------------------------
         # Change 'ignore' to 'add' to enable this configuration.
         #
+
+        rightsubnetwithin=10.1.2.0/8
```

Comment:
Has @terbolous is pointing out, could it be possible to use subnet defined in the VPC or guest network instead of hardcoding 10./8 ?

Regards,